pkg/summarizer/persist.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
nit: TestGrid Authors (here and in the other file)

Reply:
Done